### PR TITLE
Plugins {...} syntax for README

### PR DESCRIPTION
**Pull request by brunobowden**
- Move local development docs to CONTRIBUTING
- Improve CONTRIBUTING docs

### CODE REVIEW

**Comment by brunobowden** [comment]
@advayDev1, @confile - straight forward docs improvements. I've verified that the new `plugins {...}` syntax works on the published plugin.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,12 +1,59 @@
...
-5. Create a new Pull Request (once your PR has been reviewed and is ready to submit see 'Preparing your pull request for submission' below).
+2. Create your feature branch<br>
+`git checkout -b my-new-feature`
+3. Commit your changes<br>
+`git commit -am 'Add some feature'`
+4. Push to the branch<br>
+`git push origin my-new-feature`
+5. Create a new Pull Request and send it for review
+6. After the PR has been approved and is ready to submit see
+[preparing-your-pull-request-for-submission](#preparing-your-pull-request-for-submission) below).
```
Comment (advayDev1) [inline CONTRIBUTING.md]: missing start parens

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,12 +1,59 @@
...
+In order to test your modification to the plugin using your own project, you need to modify the
+project that uses the plugin:
+
+1. Comment out the `plugins {...}` section for the j2objc plugin
+1. Add in the lines to load and apply the local j2objc plugin
+
+    // File: shared/build.gradle
+    plugins {
+        id 'java'
+    // Comment out this line for local development
```
Comment (advayDev1) [inline CONTRIBUTING.md]: odd indentation

**Comment by advayDev1** [comment]
LGTM with minor comments

**Comment by confile** [comment]
LGTM

File: CONTRIBUTING.md (hunk 1 above)
Comment (brunobowden) [inline CONTRIBUTING.md]: Removed "below)"

File: CONTRIBUTING.md (hunk 2 above)
Comment (brunobowden) [inline CONTRIBUTING.md]: Fixed

**Pull request merged**